In [30]:
import numpy as np
import cv2
import pandas as pd
import os 

# import pydicom
# from pydicom import dcmread

# from fastai.basics import *
# from fastai.callback.all import *
# from fastai.vision.all import *
# from fastai.medical.imaging import *

# from pydicom.pixel_data_handlers.util import apply_voi_lut
# import pylibjpeg




# Import images from BIMVC Negative dataset.




Files are found in csv file, we have to load them and convert them to 8 bits.

- Saving CSV with all fields, it contains many information about the findings.  (in training this has to be removed)


In [48]:
def deal_filename(row):
    msg =''
    filename = row['File']
    path = path_db+'/'+row['Subject']+'/'+row['Session']+'/'+row['mod']
    img = cv2.imread( path+'/'+filename, cv2.IMREAD_ANYDEPTH)
    # convert rto 8 bits
    img = ((img / img.max()) * 255).astype('uint8')
#     img = (((img-img.min()) / (img.max()-img.min())) * 255).astype('uint8') resize the histogram?
    if row['Photometric Interpretation'] == 'MONOCHROME1':
        img = ~img
        msg = msg + 'Invert'
    
    out_fn = out_path+'/'+os.path.basename(filename)+'_8b.png'
    print (out_fn, msg)
    cv2.imwrite(out_fn,img)   
#        input("Press Enter to continue...")    

In [7]:
path_db ='/mnt/mia_images/covid/bimcv_covidN'
out_path = '/home/robert/data/covid/bimcv_covidN_db'

In [32]:
df = pd.read_csv(path_db+'/bimcv_negative.csv', header=0, sep=';')
print('len', len(df))
df.head()


len 4473


/home/robert/miniconda2/envs/fast2/lib/python3.8/site-packages/IPython/core/interactiveshell.py:3145: DtypeWarning: Columns (19,23,135,177,256) have mixed types.Specify dtype option on import or set low_memory=False.
  has_raised = await self.run_ast_nodes(code_ast.body, cell_name,


Subject     Session     mod  \
0  sub-S06039  ses-E10945  mod-rx   
1  sub-S06070  ses-E10993  mod-rx   
2  sub-S06110  ses-E11058  mod-rx   
3  sub-S06113  ses-E11062  mod-rx   
4  sub-S06116  ses-E11066  mod-rx   

                                                File Type Position_Manual  \
0  sub-S06039_ses-E10945_run-1_bp-chest_vp-pa_cr.png   cr              PA   
1  sub-S06070_ses-E10993_run-1_bp-chest_vp-ap_cr.png   cr              PA   
2  sub-S06110_ses-E11058_run-1_bp-chest_vp-pa_dx.png   dx              PA   
3  sub-S06113_ses-E11062_run-1_bp-chest_vp-ap_dx.png   dx              PA   
4        sub-S06116_ses-E11066_run-1_bp-chest_cr.png   cr              PA   

   Exclude  Exclude Reason                                             Labels  \
0    False             NaN                                          unchanged   
1    False             NaN           suboptimal study'\t 'laminar atelectasis   
2    False             NaN                                        infiltrates   
3    False             NaN  '\t 'heart insufficiency'\t ' interstitial pat...   
4    False             NaN  suboptimal study'\t 'fibrotic band'\t 'COPD si...   

                                       Localizations  ...  \
0                                                NaN  ...   
1                     loc subsegmental'\t 'loc lobar  ...   
2                             loc left'\t 'loc basal  ...   
3                loc cardiac'\t 'loc left upper lobe  ...   
4  loc right'\t 'loc bilateral'\t 'loc apical'\t ...  ...   

  Total Collimation Width Contrast/Bolus Total Dose Detector Type  \
0                     NaN                       NaN           NaN   
1                     NaN                       NaN           NaN   
2                     NaN                       0.0  SCINTILLATOR   
3                     NaN                       NaN           NaN   
4                     NaN                       NaN           NaN   

                   Imager Pixel Spacing Exposure Index  \
0                          0.143, 0.143            NaN   
1                          0.148, 0.148     870.000000   
2                          0.175, 0.175      67.560167   
3  0.138999998569489, 0.138999998569489            NaN   
4                              0.1, 0.1            NaN   

  Filter Thickness Maximum Performed Procedure Step Start Time Bits Stored  \
0                      NaN                          182705.234        12.0   
1                      NaN                                 NaN        12.0   
2                      NaN                          212816.000        12.0   
3                      NaN                                 NaN        16.0   
4                      NaN                                 NaN        12.0   

                                  Study Instance UID  \
0  1.2.840.113654.2.70.1.107288151586885502570645...   
1  1.2.840.113654.2.70.1.264783045103452617800836...   
2  1.2.840.113654.2.70.1.216859291845841333754195...   
3  1.2.840.113654.2.70.1.177717506051411834280838...   
4  1.2.840.113654.2.70.1.287550085791502471302689...   

  Patient Gantry Relationship Code Sequence  
0                                       NaN  
1                                       NaN  
2                                       NaN  
3                                       NaN  
4                                       NaN  

[5 rows x 261 columns]

In [43]:
# testing
print(df['File'].iloc[0])
deal_filename(df.iloc[0])

sub-S06039_ses-E10945_run-1_bp-chest_vp-pa_cr.png
/home/robert/data/covid/bimcv_covidN_db/sub-S06039_ses-E10945_run-1_bp-chest_vp-pa_cr.png_8b.png 


In [49]:
#  apply to all images  
df.apply(lambda row : deal_filename(row), axis = 1)

/home/robert/data/covid/bimcv_covidN_db/sub-S06039_ses-E10945_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06070_ses-E10993_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06110_ses-E11058_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06113_ses-E11062_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S06116_ses-E11066_run-1_bp-chest_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S06145_ses-E11104_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06150_ses-E12666_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04670_ses-E09155_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06170_ses-E11141_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06321_ses-E11892_run-1_bp-chest_vp-ap_dx.png_8b.png Inv

/home/robert/data/covid/bimcv_covidN_db/sub-S04544_ses-E09009_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04544_ses-E10113_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05025_ses-E09591_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05056_ses-E13167_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05056_ses-E09627_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05104_ses-E09683_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04556_ses-E12272_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04565_ses-E14123_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04565_ses-E09972_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04565_ses-E09439_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/ro

/home/robert/data/covid/bimcv_covidN_db/sub-S06625_ses-E13162_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06625_ses-E11813_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06649_ses-E11854_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06655_ses-E11864_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06656_ses-E11865_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06678_ses-E11906_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06717_ses-E11969_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06758_ses-E12025_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06761_ses-E12028_run-2_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06761_ses-E12028_run-1_bp-chest_vp-ap_cr.png_8b.p

/home/robert/data/covid/bimcv_covidN_db/sub-S08021_ses-E14252_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S08023_ses-E14259_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S08026_ses-E14264_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S08034_ses-E14277_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S08042_ses-E14289_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04856_ses-E09380_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04870_ses-E10246_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S04870_ses-E09396_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04886_ses-E09418_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S04888_ses-E10029_run-1_bp-chest_vp-ap_cr.png_8b.p

/home/robert/data/covid/bimcv_covidN_db/sub-S05628_ses-E10360_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05639_ses-E10375_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05674_ses-E13851_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05695_ses-E10459_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05701_ses-E13886_acq-1_run-1_bp-chest_vp-ll_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05701_ses-E10467_acq-1_run-1_bp-chest_vp-ll_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04615_ses-E09569_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04615_ses-E10470_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04615_ses-E09839_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04615_ses-E09145_run-1_bp-chest_vp-ap_cr.png_8b.p

/home/robert/data/covid/bimcv_covidN_db/sub-S07027_ses-E12468_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04753_ses-E13943_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04756_ses-E09264_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07077_ses-E13360_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S07077_ses-E12541_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07090_ses-E14238_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S07090_ses-E13574_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07090_ses-E12558_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07120_ses-E12599_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07146_ses-E12646_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/ro

/home/robert/data/covid/bimcv_covidN_db/sub-S05618_ses-E10338_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05627_ses-E10359_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05677_ses-E10432_acq-1_run-1_bp-chest_vp-ll_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05686_ses-E10446_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05722_ses-E10499_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05722_ses-E10499_run-2_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05746_ses-E10533_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05758_ses-E10709_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05758_ses-E11773_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05758_ses-E11773_run-2_bp-chest_vp-ap_dx.pn

/home/robert/data/covid/bimcv_covidN_db/sub-S06805_ses-E12098_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06886_ses-E12228_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06888_ses-E12237_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06889_ses-E12236_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06902_ses-E12264_run-1_bp-chest_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06903_ses-E12273_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06903_ses-E13812_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06916_ses-E12289_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04746_ses-E11204_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04746_ses-E09248_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/d

/home/robert/data/covid/bimcv_covidN_db/sub-S04953_ses-E09506_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04957_ses-E09512_run-1_bp-chest_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S04962_ses-E09520_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04989_ses-E09549_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05052_ses-E09622_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05084_ses-E11341_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05084_ses-E13225_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05098_ses-E09677_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05103_ses-E11381_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05103_ses-E12418_run-1_bp-chest_vp-ap_dx.pn

/home/robert/data/covid/bimcv_covidN_db/sub-S06045_ses-E10953_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04682_ses-E09169_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04682_ses-E10311_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06595_ses-E11767_run-1_bp-chest_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S06102_ses-E11049_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06191_ses-E11168_run-1_bp-chest_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S06670_ses-E11889_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06695_ses-E11933_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06389_ses-E11464_run-1_bp-chest_vp-pa_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S06644_ses-E11846_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/h

/home/robert/data/covid/bimcv_covidN_db/sub-S07822_ses-E13832_acq-1_run-1_bp-chest_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07822_ses-E13832_acq-2_run-1_bp-chest_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07850_ses-E13882_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07935_ses-E14039_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S07935_ses-E14128_acq-1_run-1_bp-chest_vp-ll_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07935_ses-E14128_acq-1_run-1_bp-chest_vp-ll_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07951_ses-E14086_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S04844_ses-E10715_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04844_ses-E09365_acq-1_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07988_ses-E14175_acq-1_run-1_bp-chest_vp-pa

/home/robert/data/covid/bimcv_covidN_db/sub-S04648_ses-E09126_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04654_ses-E09133_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06479_ses-E11596_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06320_ses-E11356_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04685_ses-E09542_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04685_ses-E09939_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04685_ses-E09281_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06437_ses-E11545_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06088_ses-E11024_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06267_ses-E11462_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/ro

/home/robert/data/covid/bimcv_covidN_db/sub-S07536_ses-E13343_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S07541_ses-E13350_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07664_ses-E13556_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04822_ses-E09337_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07694_ses-E14119_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07694_ses-E14022_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07837_ses-E13862_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07839_ses-E13865_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07870_ses-E13917_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07873_ses-E13922_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/ro

/home/robert/data/covid/bimcv_covidN_db/sub-S06016_ses-E14049_run-1_bp-chest_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06394_ses-E11474_run-1_bp-chest_vp-pa_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S06626_ses-E12328_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06626_ses-E11814_acq-1_run-1_bp-chest_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06413_ses-E11502_run-1_bp-chest_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S06492_ses-E13505_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06265_ses-E11274_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06132_ses-E11087_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06132_ses-E11547_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06700_ses-E11942_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/ro

/home/robert/data/covid/bimcv_covidN_db/sub-S06959_ses-E12369_acq-1_run-1_bp-chest_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06983_ses-E12404_run-1_bp-chest_vp-pa_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S06986_ses-E12410_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07000_ses-E12430_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07007_ses-E13281_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07007_ses-E12439_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07022_ses-E12460_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07030_ses-E12473_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S04757_ses-E09362_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04757_ses-E09516_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/ro

/home/robert/data/covid/bimcv_covidN_db/sub-S04919_ses-E09462_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04549_ses-E11751_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04549_ses-E12674_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S04995_ses-E10067_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05002_ses-E09562_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05002_ses-E09562_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05012_ses-E11202_acq-1_run-1_bp-chest_vp-ll_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05014_ses-E10680_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05014_ses-E09578_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05014_ses-E11229_run-1_bp-chest_vp-ap_cr.pn

/home/robert/data/covid/bimcv_covidN_db/sub-S05785_ses-E10756_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05788_ses-E10584_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05792_ses-E10588_run-1_bp-chest_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05829_ses-E10639_run-1_bp-chest_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05834_ses-E10647_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05835_ses-E12623_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05835_ses-E10648_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04628_ses-E09100_acq-1_run-1_bp-chest_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04628_ses-E09100_acq-2_run-1_bp-chest_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04628_ses-E09466_run-1_bp-chest_dx.png_8b.png 
/home/ro

/home/robert/data/covid/bimcv_covidN_db/sub-S06909_ses-E12281_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04744_ses-E09246_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06933_ses-E12322_acq-1_run-1_bp-chest_vp-ll_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06942_ses-E12338_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06963_ses-E13221_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S06963_ses-E12374_acq-1_run-1_bp-chest_vp-ll_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07015_ses-E12451_acq-1_run-1_bp-chest_vp-ll_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07018_ses-E12454_acq-1_run-1_bp-chest_vp-ll_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07019_ses-E12456_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07048_ses-E12503_run-1_bp-chest_vp-pa_dx.pn

/home/robert/data/covid/bimcv_covidN_db/sub-S04937_ses-E14261_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04546_ses-E09508_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04546_ses-E12244_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04546_ses-E11407_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04546_ses-E09345_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04546_ses-E09254_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04546_ses-E09035_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04546_ses-E09120_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04546_ses-E10517_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04546_ses-E12057_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/co

/home/robert/data/covid/bimcv_covidN_db/sub-S05732_ses-E10514_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05749_ses-E10535_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05754_ses-E10542_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05761_ses-E10551_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05761_ses-E11458_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05777_ses-E10571_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05814_ses-E10624_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05814_ses-E10620_acq-1_run-1_bp-chest_vp-ll_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05843_ses-E11926_run-1_bp-chest_vp-ap_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05843_ses-E10660_run-2_bp-chest_cr.png_8b.png Invert
/h

/home/robert/data/covid/bimcv_covidN_db/sub-S07010_ses-E12443_run-1_bp-chest_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06683_ses-E11915_acq-2_run-1_bp-chest_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06683_ses-E11915_acq-1_run-1_bp-chest_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07244_ses-E12827_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07244_ses-E13630_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S06564_ses-E11718_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06564_ses-E13063_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S07262_ses-E12855_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07288_ses-E12911_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07305_ses-E12936_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/ro

/home/robert/data/covid/bimcv_covidN_db/sub-S05358_ses-E09998_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04593_ses-E09062_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04595_ses-E11451_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04595_ses-E09417_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04595_ses-E09519_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04595_ses-E12489_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04595_ses-E09064_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04595_ses-E09773_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05473_ses-E10155_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05512_ses-E10761_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/d

/home/robert/data/covid/bimcv_covidN_db/sub-S07296_ses-E13514_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07296_ses-E12925_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06446_ses-E11556_run-1_bp-chest_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S06446_ses-E14042_run-1_bp-chest_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07004_ses-E12435_run-1_bp-chest_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S07004_ses-E12435_run-2_bp-chest_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S06133_ses-E11088_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04739_ses-E13874_run-1_bp-chest_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06294_ses-E11319_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07221_ses-E13220_run-1_bp-chest_vp-ap_cr.png_8b.png Invert
/home/robert/data/covid/bi

/home/robert/data/covid/bimcv_covidN_db/sub-S05273_ses-E09891_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05273_ses-E09897_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05317_ses-E09946_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05340_ses-E09977_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05342_ses-E09979_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05342_ses-E13076_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05342_ses-E10937_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05342_ses-E12238_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05342_ses-E11940_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05342_ses-E12436_acq-1_ru

/home/robert/data/covid/bimcv_covidN_db/sub-S07137_ses-E12627_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06922_ses-E12302_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04689_ses-E13756_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S04748_ses-E09250_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06675_ses-E11899_run-1_bp-chest_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S06632_ses-E11975_run-1_bp-chest_vp-pa_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S06632_ses-E11823_run-1_bp-chest_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S06018_ses-E10917_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S06567_ses-E11723_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06989_ses-E12414_run-1_bp-chest_vp-pa_dx.png_8b.png 
/h

/home/robert/data/covid/bimcv_covidN_db/sub-S05707_ses-E10475_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04625_ses-E09095_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04625_ses-E10675_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04625_ses-E09118_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05734_ses-E10519_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05772_ses-E10565_run-1_bp-chest_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05797_ses-E10595_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05818_ses-E11678_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05818_ses-E13019_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05818_ses-E10625_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.p

/home/robert/data/covid/bimcv_covidN_db/sub-S07500_ses-E13289_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07520_ses-E13317_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04535_ses-E09858_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04535_ses-E12707_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S04535_ses-E09000_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07629_ses-E13499_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S07629_ses-E13499_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S07654_ses-E13537_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07672_ses-E13567_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S07681_ses-E13583_run-1_bp-chest_cr.png_8b.p

/home/robert/data/covid/bimcv_covidN_db/sub-S06560_ses-E11711_run-1_bp-chest_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S06560_ses-E13280_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04806_ses-E09320_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07348_ses-E13011_run-1_bp-chest_vp-pa_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S06586_ses-E11752_acq-1_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06586_ses-E11752_acq-2_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06056_ses-E10969_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06056_ses-E12485_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06390_ses-E11466_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06074_ses-E11001_run-1_bp-chest_vp-pa_dx.png_8b.png Inv

/home/robert/data/covid/bimcv_covidN_db/sub-S04902_ses-E13623_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04956_ses-E09511_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04958_ses-E09513_acq-1_run-1_bp-chest_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04980_ses-E09539_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05085_ses-E09661_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05085_ses-E10632_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04557_ses-E12499_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05161_ses-E09751_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05183_ses-E09778_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05183_ses-E09855_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/ro

/home/robert/data/covid/bimcv_covidN_db/sub-S07592_ses-E13431_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07587_ses-E13423_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S06657_ses-E11868_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06657_ses-E13747_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06130_ses-E11085_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07122_ses-E12829_run-1_bp-chest_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S07122_ses-E12603_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07235_ses-E12815_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04713_ses-E09206_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07242_ses-E12825_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/h

/home/robert/data/covid/bimcv_covidN_db/sub-S05587_ses-E10301_acq-2_run-1_bp-chest_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05587_ses-E10301_acq-3_run-1_bp-chest_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05587_ses-E10301_acq-1_run-1_bp-chest_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05616_ses-E10336_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05616_ses-E12847_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05616_ses-E10823_acq-1_run-1_bp-chest_vp-ll_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05616_ses-E11180_acq-1_run-1_bp-chest_vp-ll_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05762_ses-E10728_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05762_ses-E10552_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05771_ses-E10564_run-1_bp-chest_vp-pa

/home/robert/data/covid/bimcv_covidN_db/sub-S06899_ses-E12260_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07281_ses-E12897_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04684_ses-E09280_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04684_ses-E09171_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06688_ses-E11923_acq-2_run-1_bp-chest_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06887_ses-E12232_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04792_ses-E09305_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07645_ses-E13524_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07652_ses-E13533_acq-1_run-1_bp-chest_vp-ll_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06734_ses-E11991_run-2_bp-chest_vp-pa_cr.png_8b.png Invert
/h

/home/robert/data/covid/bimcv_covidN_db/sub-S05310_ses-E09938_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05314_ses-E09943_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05366_ses-E10008_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05379_ses-E10022_run-1_bp-chest_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05398_ses-E10049_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05407_ses-E10061_acq-2_run-1_bp-chest_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05407_ses-E10063_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05426_ses-E10091_run-1_bp-chest_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05442_ses-E10121_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05463_ses-E10145_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.p

/home/robert/data/covid/bimcv_covidN_db/sub-S06406_ses-E11493_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S06676_ses-E11902_run-1_bp-chest_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06676_ses-E11902_run-2_bp-chest_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06509_ses-E11634_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07099_ses-E12570_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07615_ses-E13474_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06966_ses-E12377_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06966_ses-E12794_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07234_ses-E12812_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06727_ses-E12758_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/d

/home/robert/data/covid/bimcv_covidN_db/sub-S07820_ses-E13829_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07865_ses-E13909_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07869_ses-E13914_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07879_ses-E13937_run-1_bp-chest_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S07889_ses-E13954_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07905_ses-E13983_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07913_ses-E13995_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S07917_ses-E14000_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07920_ses-E14012_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S07926_ses-E14025_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/h

/home/robert/data/covid/bimcv_covidN_db/sub-S05896_ses-E10743_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05896_ses-E10743_run-2_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05946_ses-E11896_run-1_bp-chest_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05946_ses-E10816_run-1_bp-chest_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05985_ses-E10872_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06977_ses-E12394_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06977_ses-E12394_run-2_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04524_ses-E09347_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04524_ses-E08988_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04524_ses-E09036_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/co

/home/robert/data/covid/bimcv_covidN_db/sub-S07853_ses-E13887_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06519_ses-E11645_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06837_ses-E12147_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06845_ses-E12162_run-1_bp-chest_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S06845_ses-E12162_run-2_bp-chest_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S06845_ses-E12163_run-1_bp-chest_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S04973_ses-E09532_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04973_ses-E12129_run-1_bp-chest_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06463_ses-E11574_acq-3_run-1_bp-chest_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06463_ses-E11574_acq-1_run-1_bp-chest_dx.png_8b.png 
/home/robert/data/co

/home/robert/data/covid/bimcv_covidN_db/sub-S05435_ses-E10106_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05446_ses-E10125_run-2_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05446_ses-E10125_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05459_ses-E14152_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05459_ses-E10141_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05462_ses-E10144_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05491_ses-E10175_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05526_ses-E11345_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05526_ses-E10221_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05526_ses-E11063_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/co

/home/robert/data/covid/bimcv_covidN_db/sub-S07513_ses-E13308_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07821_ses-E13830_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07901_ses-E13975_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04816_ses-E09330_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06393_ses-E11473_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07570_ses-E13391_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04767_ses-E10204_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S04767_ses-E09277_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07316_ses-E12957_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06215_ses-E11200_run-1_bp-chest_vp-pa

/home/robert/data/covid/bimcv_covidN_db/sub-S06472_ses-E11587_acq-1_run-1_bp-chest_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06721_ses-E11973_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06721_ses-E11973_run-2_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07026_ses-E12467_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S06336_ses-E11380_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S07682_ses-E13584_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S06740_ses-E12001_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06414_ses-E11503_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05301_ses-E11013_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05301_ses-E12744_acq-1_run-1_bp-chest_vp-pa

/home/robert/data/covid/bimcv_covidN_db/sub-S05515_ses-E12356_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05515_ses-E10207_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05515_ses-E14011_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05515_ses-E12979_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S06454_ses-E13035_run-1_bp-chest_vp-ap_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S06454_ses-E11564_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06454_ses-E11695_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07046_ses-E12500_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07796_ses-E13792_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S08032_ses-E14275_run-1_bp-chest_vp-ap_dx.pn

/home/robert/data/covid/bimcv_covidN_db/sub-S04662_ses-E14122_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04662_ses-E10463_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04662_ses-E09832_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04662_ses-E09239_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S08012_ses-E14232_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05420_ses-E10082_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04575_ses-E09043_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04575_ses-E10161_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05063_ses-E09635_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06078_ses-E14094_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/d

/home/robert/data/covid/bimcv_covidN_db/sub-S06115_ses-E13748_run-2_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06115_ses-E11065_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07946_ses-E14074_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04540_ses-E09005_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06147_ses-E11106_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06663_ses-E11879_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07363_ses-E13037_run-1_bp-chest_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S06715_ses-E11966_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06715_ses-E13130_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06407_ses-E11984_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/d

/home/robert/data/covid/bimcv_covidN_db/sub-S06730_ses-E11985_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S07658_ses-E13544_acq-1_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05289_ses-E09909_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07123_ses-E12604_run-1_bp-chest_vp-pa_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S04745_ses-E10487_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04745_ses-E10258_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04745_ses-E12629_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05255_ses-E09870_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06343_ses-E12216_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S06343_ses-E12637_run-1_bp-chest_vp-ap_cr.pn

/home/robert/data/covid/bimcv_covidN_db/sub-S07576_ses-E13406_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07294_ses-E12920_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07294_ses-E14204_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S07294_ses-E13333_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07294_ses-E13333_run-2_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04604_ses-E09073_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04604_ses-E09469_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04934_ses-E09483_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04934_ses-E09649_acq-1_run-1_bp-chest_vp-ll_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04711_ses-E09204_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/ro

/home/robert/data/covid/bimcv_covidN_db/sub-S06425_ses-E11527_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07436_ses-E13161_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07784_ses-E13773_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07527_ses-E13817_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05226_ses-E09831_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07381_ses-E13442_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07381_ses-E13062_acq-2_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05099_ses-E09676_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04659_ses-E13867_run-1_bp-chest_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05516_ses-E10771_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/ro

/home/robert/data/covid/bimcv_covidN_db/sub-S07795_ses-E13790_acq-2_run-1_bp-chest_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07145_ses-E12643_acq-1_run-1_bp-chest_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07767_ses-E13737_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07572_ses-E13394_acq-1_run-1_bp-chest_vp-ll_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06464_ses-E11576_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04800_ses-E09312_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05415_ses-E10076_acq-1_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06449_ses-E11559_run-1_bp-chest_vp-pa_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05082_ses-E09656_acq-1_run-1_bp-chest_vp-ll_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05082_ses-E09656_acq-1_run-1_bp-chest_vp-ll_dx.pn

/home/robert/data/covid/bimcv_covidN_db/sub-S05658_ses-E10403_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05658_ses-E10403_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05690_ses-E10452_run-1_bp-chest_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05693_ses-E10456_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05702_ses-E10468_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S04621_ses-E09091_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05717_ses-E10492_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05723_ses-E10500_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05763_ses-E10553_run-1_bp-chest_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05807_ses-E10609_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/co

/home/robert/data/covid/bimcv_covidN_db/sub-S07602_ses-E13454_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07687_ses-E13592_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S04722_ses-E14188_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S04722_ses-E09215_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05529_ses-E10224_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06812_ses-E12105_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05139_ses-E09721_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06534_ses-E11666_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06072_ses-E10996_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S04592_ses-E09061_run-1_bp-chest_vp-ap_cr.png_8b.png 
/h

/home/robert/data/covid/bimcv_covidN_db/sub-S05819_ses-E10626_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05836_ses-E10651_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05860_ses-E10688_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05861_ses-E10690_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05872_ses-E10702_acq-1_run-1_bp-chest_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05893_ses-E11472_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05893_ses-E10739_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05907_ses-E10766_acq-1_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05907_ses-E10769_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05918_ses-E10783_run-1_bp-chest_vp-pa_cr.png_8b.png Invert
/h

/home/robert/data/covid/bimcv_covidN_db/sub-S05806_ses-E12971_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05165_ses-E09756_acq-1_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05165_ses-E10348_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06854_ses-E12174_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07722_ses-E13662_acq-1_run-1_bp-chest_vp-ll_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05158_ses-E12671_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05158_ses-E10364_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05158_ses-E12253_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05158_ses-E09746_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05158_ses-E10512_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/ro

/home/robert/data/covid/bimcv_covidN_db/sub-S04700_ses-E09191_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07324_ses-E13992_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07324_ses-E12970_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07793_ses-E13788_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S07941_ses-E14060_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07711_ses-E13639_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05312_ses-E09940_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05564_ses-E10272_run-1_bp-chest_vp-pa_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06964_ses-E12375_acq-1_run-1_bp-chest_vp-ll_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06104_ses-E11052_run-1_bp-chest_vp-ap_dx.png_8b.png 
/h

/home/robert/data/covid/bimcv_covidN_db/sub-S04820_ses-E09335_run-2_bp-chest_vp-ap_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S04820_ses-E09335_run-1_bp-chest_vp-ap_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05509_ses-E10199_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S07904_ses-E13982_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S06710_ses-E12701_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06710_ses-E12475_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06710_ses-E14187_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06710_ses-E11957_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S08033_ses-E14276_run-1_bp-chest_cr.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S04591_ses-E09060_run-1_bp-chest_vp-ap_cr.png_8b.p

/home/robert/data/covid/bimcv_covidN_db/sub-S07038_ses-E12487_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07204_ses-E12759_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07204_ses-E12759_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07204_ses-E12759_acq-1_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S07204_ses-E13850_run-1_bp-chest_vp-ap_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S06513_ses-E11639_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05381_ses-E10026_run-1_bp-chest_vp-pa_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S06339_ses-E11384_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S06803_ses-E12095_run-1_bp-chest_vp-ap_cr.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05038_ses-E09608_run-1_bp-chest_vp-ap_cr.pn

/home/robert/data/covid/bimcv_covidN_db/sub-S05638_ses-E10374_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05299_ses-E09924_run-1_bp-chest_vp-ap_dx.png_8b.png 
/home/robert/data/covid/bimcv_covidN_db/sub-S05994_ses-E10886_run-1_bp-chest_vp-pa_dx.png_8b.png Invert
/home/robert/data/covid/bimcv_covidN_db/sub-S05998_ses-E10890_run-1_bp-chest_vp-ap_dx.png_8b.png Invert


0       None
1       None
2       None
3       None
4       None
        ... 
4468    None
4469    None
4470    None
4471    None
4472    None
Length: 4473, dtype: object

## Get Train and test stratified at a patient level.

Stratifying at patient level, there are 3098 patients and 4473 total images, we should be careful to stratify correctly (no patient goes in both test and train)


In [63]:
repeatdf = df[df.duplicated('File')]
print('Total of repeated filenames', len(repeatdf))
# print(repeatdf['File'])

Total of repeated filenames 58


In [69]:
# trn,val = RandomSplitter(valid_pct = 0.2, seed=42)(items)

print("images ",len(df))
df_patients= df['Subject'].unique()
print("Patients ",len(df_patients))

msk = np.random.rand(len(df_patients)) < 0.8
df_pa_train = df_patients[msk]
df_pa_test = df_patients[~msk]


df_train = df.loc[df['Subject'].isin(df_pa_train)]

df_test = df.loc[df['Subject'].isin(df_pa_test)]
print('Train: ',len(df_train), 'Test',len(df_test))
# df_test.head()

images  4473
Patients  3098
Train:  3594 Test 879


## Saving the csv

In [68]:


df_train.to_csv(out_path+'/bimcv_neg_train.csv')
df_test.to_csv(out_path+'/bimcv_neg_test.csv')